pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -43,6 +43,14 @@ const builtinRegistriesConfDirPath = "/etc/containers/registries.conf.d"
 // helper.
 const AuthenticationFileHelper = "containers-auth.json"
 
+type PullFromMirror string
+
+const (
+	MirrorAll          PullFromMirror = "all"
+	MirrorByDigestOnly PullFromMirror = "digest-only"
+	MirrorByTagOnly    PullFromMirror = "tag-only"
+)
```

Comment:
Do we really need to export them?  I'd expect a simple string (no extra type).

Reply:
> Could you add comments to them? I am still not sure whether introducing a new type is needed though.

yes, the new type is not necessary. Removed the new type.